blueocean-core-js/src/js/sse/SseBus.js

```diff
@@ -44,7 +45,7 @@ export class SseBus {
         if (!this.sseListeners.job) {
             const sseListener = this.connection.subscribe('job', (event) => {
                 this._handleJobEvent(event);
-            });
+            }, { jenkins_org: AppConfig.getOrganizationName() });
```

Comment:
Each UI instance knows one org. All we are telling is backend SSE pub-sub to only return events for this org. So in multi-master case, there can be special pub sub implementation that would filter events accordingly. Why you think scalability could be an issue?